omod/src/main/webapp/pages/htmlform/editHtmlFormWithSimpleUi.gsp

```diff
@@ -80,10 +80,10 @@ ${ ui.includeFragment("htmlformentryui", "htmlform/enterHtmlForm", [
 ]) }
 
 <script type="text/template" id="confirmation-template">
-<div id="confirmation">
+<div id="confirmation" class="container">
     <span class="title">${ ui.message("coreapps.simpleFormUi.confirm.title") }</span>
 
-    <div id="confirmationQuestion" class="container">
```

Comment:
Not sure why the container was on the confirmation question (and I think this "container" class predated Bootstrap) but moving it up fixes the layout of the confirmation questions on a tablet (or at least in the testing I've done).

Reply:
If you have a minute, it would be great to see where in the CSS that container class is defined and used. It would be great not to have namespace collision with Bootstrap since Bootstrap is being used, if its an easy fix.

Reply:
Fair enough... I did a quick check but I can't really track it down anywhere... looking in the dev tools I'm only seeing styling coming in from _breakpoints.scss and _code.scss, which I think might actually be Bootstrap classes?  In any case, we dont' seem to define "container" in OpenMRS custom scss?